src/Sql/Sql/Common/Constants.cs

```diff
@@ -32,5 +32,10 @@ public class Constants
         public const string ComputeGenerationGen5 = "Gen5";
         public const string CollationSqlLatin1 = "SQL_Latin1_General_CP1_CI_AS";
         public const string CollationLatin1 = "Latin1_General_100_CS_AS_SC";
+        public const string PublicDataEndpointEnabledTrue = "$true";
+        public const string PublicDataEndpointEnabledFalse = "$false";
+        public const string ProxyOverrideDefault = "Default";
+        public const string ProxyOverrideProxy = "Proxy";
+        public const string ProxyOverrideRedirect = "Redirect";
```

Comment:
Doesn't .net sdk have generated constant that you can use instead of redefining?

Reply:
Yes, it does. I'll update this.

Reply:
Updated.